Xamarin.Forms.Platform.iOS/Renderers/ImageRenderer.cs

```diff
@@ -246,7 +246,7 @@ public async Task<FormsCAKeyFrameAnimation> LoadImageAnimationAsync(ImageSource
 	public sealed class FontImageSourceHandler : IImageSourceHandler
 	{
 		//should this be the default color on the BP for iOS? 
```

Comment:
Should it?

Reply:
No, removed the comment for two reasons:
- If the BP is the default color, we will set the default color that is determined here. Which will now be the OS theme aware color. So that will be white or black, just like the text. IMO doesn't make sense to add logic for that in the core when it's handled here.
- No one complained so I guess it works as it should :P